catroid/src/main/java/org/catrobat/catroid/content/bricks/ThinkForBubbleBrick.java

```diff
@@ -95,20 +92,12 @@ public View getView(Context context, int brickId, BaseAdapter baseAdapter) {
 				.brick_think_for_bubble_edit_text_text;
 		int editDurationId = type == Constants.SAY_BRICK ? R.id.brick_say_for_bubble_edit_text_duration : R.id
 				.brick_think_for_bubble_edit_text_duration;
-		int thinkSaySecondsLabelId = type == Constants.SAY_BRICK ? R.id.brick_say_for_bubble_seconds_label : R.id
-				.brick_think_for_bubble_seconds_label;
+		int thinkSaySecondsLabelId = type == Constants.SAY_BRICK ? R.id.brick_say_for_bubble_seconds_text_view : R.id
+				.brick_think_for_bubble_seconds_text_view;
```

Comment:
why did the other ids changed from _text_view to _ label and this one the other way around?

Reply:
please resolve the conflicts

Reply:
My first approach with the icons was to add them to the TextView with the '_label' keyword. The ThinkForBubbleBrick however, had two TextViews with the '_label' keyword, so it had two icons. To resolve this i removed the '_label' from the second TextView.
Since then I changed it, so that it just adds the icon to the first TextView instead of looking for the keyword. 
So now it doesn't matter anymore if it has a '_label' in it or not, but for consistency's sake it doesn't hurt for the first TextView to have that keyword, as most TextViews in the Bricks have it. I can of course change it back again if you wish.

Conflicts resolved.